cloudinit/util.py

```diff
@@ -68,6 +68,10 @@
                    ['lxc-is-container'])
 
 
+def kernel_version():
+    return tuple(map(int, os.uname().release.split('.')[:2]))
```

Comment:
we don't have release anywhere below?

Reply:
I tihnk we have distro release handling, nor kernel version. We could possibly map one to other, but I think it's a little more complicated than just grabbing kernel version directly.